src/formatters/__tests__/fixtures/b.css

```diff
@@ -0,0 +1 @@
+a { top: 2.0; }
```

Comment:
I think there is no need for the fixtures. We are not testing that stylelint is working correctly, just that the formatters are.